diff -urNp -x '*.orig' netkit-rwho-0.17.org/rwhod/rwhod.c netkit-rwho-0.17/rwhod/rwhod.c
--- netkit-rwho-0.17.org/rwhod/rwhod.c	2026-03-03 00:27:48.876429431 +0100
+++ netkit-rwho-0.17/rwhod/rwhod.c	2026-03-03 00:27:48.942709372 +0100
@@ -699,6 +699,11 @@ configure(int s)
 			continue;
 		}
 		strcpy(np->n_name, ifr->ifr_name);
+		if (strchr(np->n_name, ':')) {
+		  /* virtual interface found */
+		  free((char *) np);
+		  continue;
+		}
 		np->n_addrlen = sizeof (ifr->ifr_addr);
 
 		np->n_dstaddr = malloc(np->n_addrlen);
